Makefile

```diff
@@ -14,9 +14,14 @@ modified_only_fixup:
 		echo "No library .py files were modified"; \
 	fi
 
+# Update src/transformers/dependency_versions_table.py
+
+deps_table_update:
+	@python setup.py deps_table_update
```

Comment:
Why the @ ?

Reply:
it doesn't echo the command line if you're asking what @ does. There are already 2 lines the distutils cmd emits, one useless 'running  deps_table_update' I can't control, one informative 'updating src/transformers/dependency_versions_table.py'

```
running deps_table_update
updating src/transformers/dependency_versions_table.py
```

If we remove @, we will have:

```
python setup.py deps_table_update
running deps_table_update
updating src/transformers/dependency_versions_table.py
```

I wish I could kill `running deps_table_update` and leave just the last one.

Reply:
Thanks for explaining, I was just wondering what its use was.